tests/db/jobs/test_jobcontroller.py

```diff
@@ -1112,6 +1112,15 @@ def test_running_runner(job_controller, daemon_manager, wait_daemon_started):
     assert job_controller.get_running_runner() == "NO_DOCUMENT"
 
 
+def test_runner_pings(job_controller):
+    data = {"test": "XX"}
+    job_controller.ping_running_runner(data=data)
+    job_controller.ping_running_runner(data=data)
+    assert len(job_controller.get_runner_pings()) == 2
+    job_controller.clean_runner_pings()
+    assert len(job_controller.get_runner_pings()) == 0
```

Comment:
I'd make a check that it does not go beyond the max number of pings (and also that when it reaches the max number, it is the oldest one that gets kicked). That means 50 pings, but I also suggested to put the 50 as a class variable that can then be used to dynamically change it here and make the test.